src/Core/Framework/Adapter/Cache/SymfonyCacheClearerAdapter.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Cache;
+
+use Shopware\Core\Framework\Adapter\Cache\CacheClearer as ShopwareCacheClearer;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Component\HttpKernel\CacheClearer\CacheClearerInterface;
+
+#[Package('core')]
+class SymfonyCacheClearerAdapter implements CacheClearerInterface
+{
+    public function __construct(
+        private readonly ShopwareCacheClearer $cacheClearer
+    ) {
+    }
+
+    public function clear(string $cacheDir): void
+    {
+        $this->cacheClearer->clear();
```

Comment:
You could add a optinal parameter like "$clearAdapterCache" to the ShopwareCacheClearer and set it to "false" here.

Reply:
Ok, I instroduce this parameter to prevent adapter cache to be cleared.